app/models/event.rb

```diff
@@ -17,4 +18,22 @@ def before_status_full
   def after_status_full
     STATUSES[after_status]
   end
+
+  def new_for?(user)
+    notifications.where('created_at > ? and user_id = ?',
```

Comment:
Because i this method checks that user seen some event.(I mark unread events in feed as in facebook notifications.)